client/src/pages/singlePlace/FakeData.js

```diff
@@ -0,0 +1,8 @@
+const singleSitSpot={
+  name: "Dkhhkj jlkhl", 
+  country: "UK", 
+  city: "London", 
+  website: "http://hkjhk.com",
+}
+
+export default singleSitSpot;
```

Comment:
is your eslint working?

Reply:
No, it didn't work yesterday but today is working ^_^